ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto

```diff
@@ -20,8 +20,6 @@ message Completion {
   string command_id = 1;
 
   // Identifies the exact type of the error.
-  // For example, malformed or double spend transactions will result in a ``INVALID_ARGUMENT`` status.
-  // Transactions with invalid time time windows (which may be valid at a later date) will result in an ``ABORTED`` error.
```

Comment:
I would redirect the user to the Ledger API error codes for reference and apply the same pattern for `InterfaceView.view_status`

Reply:
changed